Formula/kde-extra-cmake-modules.rb

```diff
@@ -0,0 +1,28 @@
+class KdeExtraCmakeModules < Formula
+  desc "Extra modules and scripts for CMake"
+  homepage "https://api.kde.org/frameworks/extra-cmake-modules/html/index.html"
+  url "https://download.kde.org/stable/frameworks/5.70/extra-cmake-modules-5.70.0.tar.xz"
+  sha256 "830da8d84cc737e024ac90d6ed767d10f9e21531e5f576a1660d4ca88bee8581"
+
+  head "git://anongit.kde.org/extra-cmake-modules"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "qt" => :build
+  depends_on "sphinx-doc" => :build
+
+  def install
+    args = std_cmake_args
+    args << "-DBUILD_HTML_DOCS=ON"
+    args << "-DBUILD_MAN_DOCS=ON"
+    args << "-DBUILD_QTHELP_DOCS=ON"
+    args << "-DBUILD_TESTING=OFF"
+
+    system "cmake", ".", *args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"CMakeLists.txt").write("find_package(ECM REQUIRED)")
+    system "cmake", ".", "-Wno-dev"
```

Comment:
Will this generate some output we can check?

Reply:
```
expected="ECM_DIR:PATH=#{HOMEBREW_PREFIX}/share/ECM/cmake"
assert_includes File.read(testpath/"CMakeCache.txt"), expected
```

is this looks legit?

Reply:
Use assert_match, flip the arguments an you're good to go I think.